gui/gui.md

```diff
@@ -97,3 +97,52 @@ The "Advanced Preferences" section can be ignored by most users. Selecting "Deve
 ### Email Server
 
 The final tab, "Email Server", is extremely important for users who intend to use Gravwell to send automated email alerts.  Complete documentation is available on the [Email Configuration](/configuration/email) page.
+
+## Systems & Health Menu
+
+The Systems & Health sub-menu contains pages which describe the current state of the Gravwell cluster.
+
+![](systems-and-health.png)
+
+### Storage, Indexers, & Wells
+
+This page shows information about the data stored in the indexers of the Gravwell system.
+
+![](storage.png)
+
+The Storage section shows a summary of how much data is in the system, with separate stats for hot and cold storage. The "dropper" graphic indicates how fast new entries are entering the system.
```

Comment:
"The Storage section shows a summary of how much data is in the system, with separate stats for hot and cold storage"
Do we need the comma before `with`?

Reply:
i think it's correct